blog/content/post/2019-11-22-Websummit-2019.md

```diff
@@ -0,0 +1,107 @@
+---
+title: '3mdeb on Web Summit 2019'
+abstract: 'Web Summit 2019 - the biggest tech event in the World.
+          Take a short brake and read about our journey and impressions.'
+cover: /covers/web_summit_1.jpg
+author: lukasz.wcislo
+layout: post
+published: true
+date: 2019-11-22
+archives: "2019"
+
+tags:
+  - Web Summit
+  - Conference
+categories:
+  - Miscellaneous
+
+---
+
+Founded in 2009, originally held in Dublin until 2016, when it moved permanently
+to Lisbon, Web Summit is considered to be the largest tech event in the world.
+Until 2019 there was no representation of 3mdeb on Web Summit, but now we've
+decided, that it is the right time to send one.
+
+"Tech event" is the keyword that turns all of us on and makes us travel over
+thousand kilometers, but after the struggle, I was the one who becomes the first
+official 3mdeb representative on that event. Full of hope and curiosity, with
+a great financial and organizational support provided by
+[Pomeranian Export Broker](http://www.brokereksportowy.pl), at Sunday, November
+3, I've joined Polish group on the Lech Walesa Airport in Gdansk.
+
+#### Have you heard about the main types of a lie?
+
+In the classical version, there are small lies, big lies, and statistics. I've
+found another - calling Web Summit a tech event. Blows up the scale. It was
+political, it was sociological, it was futuristic, but it has much more to say
+about the people relations, than about technology.
+
+The opening day was a series of presentations. The main guest wasn't physically
+with us, but his speech was streamed from Russia. Edward Snowden - the great
+warrior for freedom of speech, which had left the western world and settled down
+in the capital of freedom - Moscow. Though I don't consider him the most
+reliable man living, the speech itself was fine. In the end, he started to
+accuse companies by their name for trying to enslave the world population, and
+when he got to Huawei it appeared that he runs out of time. The next speaker was
+Guo Ping, Huawei's Rotating Chairman - astonishing coincidence, isn't it?
+He tried to persuade an audience that all the things anyone heard about his
+company are a lie, no matter the facts. Hopefully, there were a lot of young
```

Comment:
are lies? or is lie?

Reply:
@pietrushnic "is a lie" would be incorrect. "Are lies" are correct - I'll use this. But, to my surprise, "are a lie" is correct too. Plural form refers to "all the things" and singular to an object - a lie - formed by them.